src/main/java/com/cloudbees/jenkins/plugins/amazonecs/ECSCloud.java

```diff
@@ -276,6 +295,17 @@ public String getDisplayName() {
             return Messages.DisplayName();
         }
 
+        public ListBoxModel doFillAwsRegionItems() {
```

Comment:
Amazon recommend to query the API for available regions, not use a hard-coded enum. See [ec2-plugin](https://github.com/jenkinsci/ec2-plugin/blob/6b286b185ba41efc33ab8558a6c43969975e6238/src/main/java/hudson/plugins/ec2/AmazonEC2Cloud.java#L158) for sample implementation.